news.js

```diff
@@ -0,0 +1,247 @@
+}
+
+function footer() {
+	let html = `<p class="footer__copyright">&copy; NewsFeed 2019</p>`;
+	document.getElementById('footer').innerHTML = html;
+}
+
+function displayForm() {
+	var form = document.createElement('div');
+	form.setAttribute('id', 'iamform');
+	let main = document.getElementById('main');
+	main.appendChild(form);
+
+	var allChannels = '';
+	for (let i = 0; i < channels.length; i++) {
+		allChannels += `<option value='` + channels[i] + `'>` + channels[i] + `</option>`;
+	}
+
+	document.getElementById('iamform').innerHTML =
+		`<div class='form'>
+		<label for='sel-category' class='form__select-label'><b>SELECT CATEGORY</b></label>
+		<select id='sel-category' class='form__select-box' onchange='myFunction()'>
+		// If the select is changed myFunction() is called
+		<option value='All' selected>All</option>` +
+		allChannels +
+		`</select>
+		<label for='subscribe' class='form__subscribe-label'><b>SUBSCRIBE</b></label>
+		<input type='text' id='subscribe' class='form__subscribe-textbox' placeholder='Email Address'/>
+		<button class='form__subscribe-button' onclick='validate()'>Subscribe</button>
+		</div>`;
+}
+
+function displayNews() {
+	var disp_news = document.createElement('div');
+	disp_news.setAttribute('id', 'displaynews');
+	let main = document.getElementById('main');
+	main.appendChild(disp_news);
+
+	for (let i = 0; i < data.length; i++) {
+		fulldata +=
+			`<div class='content' id='content'>
+			<div class='content__sub' id='content__display'>` +
+			data[i].image +
+			`<h3 class='content__modifier content__head' id='myBtn'>` +
+			data[i].heading +
+			`</h3>
+			<p class='content__modifier content__date'>` +
+			data[i].date +
+			`</p>
+			<p class='content__modifier content__matter'>` +
+			data[i].content +
+			`</p>
+			<a href='#!' class='content__modifier btn btn--pink' id='myBtn' 
+			onclick='showpopup(` +
+			i +
+			`)'>Continue Reading</a>
+			</div>
+			<hr>`;
+	}
+
+	//Appending model data to fulldata
+	fulldata += modal_data;
+	document.getElementById('displaynews').innerHTML = fulldata;
+}
+
+//For Validation of Email Address
+var email = [];
+function validate() {
+	if (/^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$/.test(document.getElementById('subscribe').value)) {
+		let loc = localStorage.getItem('iamkey');
+		if (loc) {
+			email = JSON.parse(loc);
+		}
+		email.push(document.getElementById('subscribe').value);
+		localStorage.setItem('iamkey', JSON.stringify(email));
+		console.log(localStorage.getItem('iamkey'));
+		alert('You have entered a valid email address!');
+		return true;
+	}
+	alert('You have entered an invalid email address!');
+	return false;
+}
+
+//When the user clicks the button, open the modal (for one news channel)
+function showpopup(i) {
+	document.getElementById('popup_head').innerHTML = data[i].heading;
+	document.getElementById('popup_content').innerHTML = data[i].popup;
+	let modal = document.getElementById('myModal');
+	modal.style.display = 'block';
```

Comment:
You can use modal.classList.add('displayBlock')
modal.classList.remove('displayBlock')